zhmccli/_cmd_partition.py

```diff
@@ -162,6 +165,25 @@ def partition_stop(cmd_ctx, cpc, partition):
 @click.option('--boot-media-file', type=str, required=False,
               help='Boot from removable media on the HMC: The path to the '
               'image file on the HMC.')
+@click.option('--type', type=click.Choice(PARTITION_TYPES), required=False,
+              help='Defines the type of the partition (Default: {pd}).'
+              .format(pd=DEFAULT_PARTITION_TYPE))
+@click.option('--ssc-host-name', type=str, required=False,
+              help='Secure Service Container host name.')
+@click.option('--ssc-boot-selection',
+              type=click.Choice(['appliance', 'installler']), required=False,
+              help='Defines whether to run the Secure Service Container '
+              'appliance installer or the Secure Service Container appliance '
+              'itself (Default: {bd}).'
+              .format(bd=DEFAULT_SSC_BOOT))
```

Comment:
This cannot be selected when creating the partition. Please remove the option from the create subcommand.